extensions-core/protobuf-extensions/src/test/java/org/apache/druid/data/input/protobuf/ProtoTestEventWrapper.java

```diff
@@ -360,13 +360,13 @@ private ProtoTestEvent(
     public static com.google.protobuf.Descriptors.Descriptor
     getDescriptor()
     {
-      return org.apache.druid.data.input.protobuf.ProtoTestEventWrapper.internal_static_prototest_ProtoTestEvent_descriptor;
+      return org.apache.druid.data.input.protobuf.ProtoTestEventWrapper.INTERNAL_STATIC_PROTOTEST_PROTOTESTEVENT_DESCRIPTOR;
```

Comment:
`ProtoTestEventWrapper` is a generated class. Don't edit it. Please add an exclusion to the Checkstyle rule.

Reply:
I did not understand this! shouldn't I make changes to this

Reply:
https://github.com/apache/incubator-druid/blob/abf9843e2a217656c4fa4bc79ffd771b13680281/extensions-core/protobuf-extensions/src/test/java/org/apache/druid/data/input/protobuf/ProtoTestEventWrapper.java#L20-L26

Reply:
Yes, do not edit.